extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -148,11 +148,11 @@ public function resetFailedTransfers(){
 			'city_visualization_images',
 			array(
 				'reviewer_id = null',
-				'image_review_status' => ImageReviewStatuses::STATE_UNREVIEWED,
+				'image_review_status' => ImageReviewStates::UNREVIEWED,
 			),
 			array(
 				"review_start < now() - " . self::INVALID_STATUS_TIMEOUT,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "review_start < now() - " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)